src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -175,9 +193,12 @@ struct convert<LayerVariantAttr<T>> {
     }
 };
 
+static const std::set<std::string> set_UniformGenerator_parameters = {"low", "high"};
```

Comment:
please don't use `set` in the name and align names with code style of the file

Reply:
Please don't do this global variables with internal linkage, just put the into the anonymous namespace

Reply:
Done